pages/common/java.md

```diff
@@ -11,6 +11,10 @@
 
 `java -jar {{filename.jar}}`
 
+- Execute a .jar program with debug waiting to connect on port 5005: 
```

Comment:
I think this is a fairly advanced usage of the command, which unfortunately makes it out of scope for a tldr page.

Reply:
its the one thing you would need when using it from command line.

perl and gcc has info on how to do use and enable debug. 

unfortunately in java the line is just quite long and verbose; but thats why it qualifies to be in tldr as noone remembers that by heart imo.

Reply:
/cc @agnivade - I think this one has slipped through the net

Reply:
Well, I think I still respectfully disagree that this needs to be added. But if you think that this is fine, then I have no objections.